lib/utils/sendNotifications.js

```diff
@@ -33,6 +39,120 @@ const sendArticleNotificationToAuthorFollowers = async (id, title) => {
 };
 
 
+const sendNotificationToFollowedUser = async (id, followeeId) => {
+  const UserToNotify = await User.getUser(id);
+  const followerDetails = await User.getUser(followeeId);
+  if (UserToNotify.dataValues.userprofile.dataValues.emailnotification === true) {
```

Comment:
```suggestion
  if (UserToNotify.userprofile.emailnotification === true) {
```
You don't have to use `dateValues` you can perform a direct property lookup on the instance object.

Reply:
I have fixed this rex...